server/src/sim/financialanalysis.py

```diff
@@ -48,7 +47,7 @@ def financialanalysis(D, postyear=2015, S=None, rerunmodel=False, artgrowthrate=
 
     # Get most recent ART unit costs 
```

Comment:
@Ashalynd i have been thinking about this - it won't be an issue in practice because every country has an ART program, but at some point we will change the way that this is done - probably ART won't be treated as a program in the same way as the other programs.